src/com/nimbella/sdk/Connection.java

```diff
@@ -0,0 +1,89 @@
+package com.nimbella.sdk;
+
+import java.sql.DriverManager;
+import com.google.cloud.storage.Storage;
+import com.google.cloud.storage.StorageOptions;
+import com.google.gson.Gson;
+import com.google.gson.JsonObject;
+
+import redis.clients.jedis.Client;
+import redis.clients.jedis.Jedis;
+import redis.clients.jedis.JedisPoolConfig;
+
+public class Connection {
+
+	public static Client redisConnect() throws Exception {
+		String redisHost = System.getenv().get("__NIM_REDIS_IP");
+		if(redisHost.isEmpty() || redisHost == null) {
```

Comment:
null check should come first.
```suggestion
		if(redisHost == null || redisHost.isEmpty()) {
```